CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing
+
+First off, thanks for taking the time!
+If you are interested in contributing to Opta, we'd love to hear from you! Drop us a line in our [Slack](https://slack.opta.dev/).
+
+## Orientation
+
+Here's a list of repositories that contain Opta-related code:
+
+- [opta](https://github.com/run-x/opta)
+  is the main repository containing the `Opta` core code and modules.
+- [opta-docs](https://github.com/run-x/opta-docs) contains all the documentation and the Opta installation script.
+
+## Types of Contributions
+
+### Report Bug
+
+The best way to report a bug is to file an issue on GitHub. Please make sure there is an open issue discussing your contribution. Before opening a new issue, please check for [existing issues](https://github.com/run-x/opta/issues). If you find an existing issue that matches closely with yours, please thumbs-up or comment on it, so we know that the issue is relevant to many people. For any new issue please include:
+
+- Your operating system name and version.
+- Opta version.
+- Detailed steps to reproduce the bug.
+- Any details about your local setup that might be helpful in troubleshooting.
+- When posting Python stack traces, please quote them using
+[Markdown blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/).
+- Tag the issue with `bug`
+- Remember that this is a volunteer-driven project, and that contributions are welcome :)
```

Comment:
all this information and some boiler plate can be in a issue template
ex: https://github.com/DataDog/datadog-agent/blob/main/.github/ISSUE_TEMPLATE.md

Reply:
We already have issue templates: https://github.com/run-x/opta/tree/main/.github/ISSUE_TEMPLATE 

I am just repeating things from there. Helps folks who haven't created an issue yet, on what the process would look like when they do.

Reply:
cool, sorry I forgot to check